components/login/login.html

```diff
@@ -1,26 +1,25 @@
 
 <div class="container">
-  <div data-bind="visible: status() == 'loading'">
-  </div>
+  <div data-bind="visible: status() == 'loading'"></div>
 
   <div class='login col-lg-6' data-bind="visible: status() == 'login'">
     <h1>Login</h1>
     <!-- ko if: loginError -->
-    <div class="loginError" data-bind="text: loginError"></div>
+    <div class="alert alert-danger loginError" role="alert" data-bind="text: loginError"></div>
     <!-- /ko -->
-    <form data-bind="submit: login" role="form">
+    <form data-bind="submit: login">
 
       <div class="form-group">
         <label for="inputUsername">Username</label>
-        <input type="text" class="form-control" id="inputUsername" placeholder="Username" data-bind="value: username">
```

Comment:
Removing these `type="text"` attributes breaks the clicktests. ([Example](https://github.com/FredrikNoren/ungit/blob/e6fa906db2a7b1c8a8bb4038294bb36f9fe94a75/nmclicktests/spec.authentication.js#L17-L31))
Was there a particular reason to remove them other then relying on default browser behavior?
I personally like to be explicit in these cases especially for `<button>` `type`s.

Reply:
Yes, mostly to reduce the number of attributes.

Reply:
Reverted the removed `type`s.